src/intersection-observer.js

```diff
@@ -219,6 +219,10 @@ export class IntersectionObserver extends Observable {
    * @private
    */
   flush_() {
+    if (!this.iframe_) {
+      return;
```

Comment:
This wouldn't actually work. I think we should create a `destroy` function and call it in all places that set intersectionObserver to null. This function can then call `clearTimeout` to make this problem go away.